src/app/head/reducer.spec.ts

```diff
@@ -3,6 +3,9 @@ import reducer, { initialState } from './reducer';
 
 describe('head reducer', () => {
   const action = setHead({
+    link: [
```

Comment:
should this be `links` plural?

Reply:
I set it to be singular because `meta` was singular and I thought `meta` and `link` referred to the element name.

I was also confused when I saw `meta` as singular, so making them both plural makes sense to me.